alpha-build-machinery/make/targets/openshift/crd-schema-gen.mk

```diff
@@ -4,23 +4,27 @@ CRD_SCHEMA_GEN_APIS ?=$(error CRD_SCHEMA_GEN_APIS is required)
 CRD_SCHEMA_GEN_MANIFESTS ?=./manifests
 CRD_SCHEMA_GEN_OUTPUT ?=./manifests
 
-crd_patches :=$(subst $(CRD_SCHEMA_GEN_MANIFESTS),$(CRD_SCHEMA_GEN_OUTPUT),$(wildcard $(CRD_SCHEMA_GEN_MANIFESTS)/*.crd.yaml-merge-patch))
```

Comment:
> Using the global variable, the temp directory for CRD_SCHEMA_GEN_OUTPUT wasn't properly passed through to update-codegen, meaning the patches didn't get applied to the temp dir

`s/:=/=/`

Reply:
my bad, but if it was failing and you are fixing it make sure to extend the test suite to catch it